proposals/low-level-struct-improvements.md

```diff
@@ -285,7 +284,9 @@ Span<int> Use()
 }
 ```
 
-The span safety rules will be written in terms of `scoped ref` and `ref`. For span safety purposes an `in` parameter is equivalent to `ref` and `out` is equivalent to `scoped ref`. Both `in` and `out` will only be specifically called out when it is important to the semantic of the rule. Otherwise they are just considered `ref` and `scoped ref` respectively.
+The span safety rules will be written in terms of `scoped ref` and `ref`. For span safety purposes an `in` parameter is equivalent to `ref` and `out` is equivalent to `scoped ref`. Both `in` and `out` will only be specifically called out when it is important to the semantic of the rule. Otherwise they are just considered `ref` and `scoped ref` respectively. 
+
+When discussing the *ref-safe-to-escape* of arguments that correspond to `in` parameters they will be generalized as `ref` arguments in the spec. In the case the argument is an lvalue then the *ref-safe-to-escape* is that of the lvalue, otherwise it is *current method*. Again `in` will only be called out here when it is important to the semantic of the current rule.
```

Comment:
I continued to be frustrated at having to call `in` out differently than `ref` in a couple of places. Hoping to fix that by clarifying that the *ref-safe-to-escape* of `in` changes based on whether it's an lvalue or compiler generated temp. Allowed me to remove `in` language from a few places.

Reply:
I didn't understand why we needed a distinction for a compiler-generated temp. It's just another local variable, right?

Reply:
The spec often refers to the *ref-safe-to-escape* of the `ref` argument. In the case we're referring to `in` though it could be the expression or the compiler generated temp. I think it's understood it's 99% talking about "what we actually pass" vs. "what is actually typed in the editor" but it's a spec so need to be specific.